packages/playwright/src/program.ts

```diff
@@ -201,6 +202,19 @@ async function runTests(args: string[], opts: { [key: string]: any }) {
   gracefullyProcessExitDoNotHang(exitCode);
 }
 
+function computeCommandHash(args: string[], opts: { [key: string]: any }) {
+  const command = {
+    args,
+    opts: {} as any
+  };
+  // Omit shard from the hash, shard number is added explicitely at the end of the blob report file name.
+  const keys = Object.keys(opts).filter(k => k !== 'shard');
```

Comment:
nit: i looked at the options and we don't seem to care about most of them. So maybe only use those definding the scope of the run? `[args], { grep, grepInvert, project, shard }`